src/MatrixClient.ts

```diff
@@ -897,6 +898,34 @@ export class MatrixClient extends EventEmitter {
         }
     }
 
+    /**
+     * This returns a list of message and state events for a room.
+     *
+     * It uses pagination query parameters to paginate history in the room.
```

Comment:
yes, that's how pagination works :p

Possibly something like this for the overall description?

> Retrieve a (paginated) chunk of history from the room. The direction in which the server is
> asked to find events in can be changed, with `b` (backwards) being "find events older than
> the provided token".

We really shouldn't be copy/pasting from the spec, particularly when it doesn't make sense.